lib/postgrex/protocol.ex

```diff
@@ -964,6 +1021,8 @@ defmodule Postgrex.Protocol do
     do: {:commit, nil}
   defp decode_tag("ROLLBACK"),
     do: {:rollback, nil}
+  defp decode_tag( :suspended ),
+    do: {:select, nil}
```

Comment:
a run 

```
iex(12)> Postgrex.stream( pid, q, [], max_rows: 1 ) |> Enum.to_list()

** (Dbg) #PID<0.157.0> calls Postgrex.Protocol.decode_tag/1 with arguments:
    [:suspended]

** (Dbg) #PID<0.157.0> calls Postgrex.Protocol.decode_tag/1 with arguments:
    [:suspended]

** (Dbg) #PID<0.157.0> calls Postgrex.Protocol.decode_tag/1 with arguments:
    ["SELECT 0"]
[%Postgrex.Result{columns: ["price"], command: :select, connection_id: 4646,
  num_rows: 1, rows: [[34.99]]},
 %Postgrex.Result{columns: ["price"], command: :select, connection_id: 4646,
  num_rows: 1, rows: [[6.99]]}]
```


return `{:select, nil}` leverages existing code https://github.com/ericmj/postgrex/pull/176/files#diff-3e2400d928ee8dfffbe041e24515a8b2R800 to fill in the `num_rows`...

complete( stream ) calls complete( query ) which is not tailored to receive a empty tag ...

an alternate I think is `complete( stream )` does not call `complete( query )` and pass to `sync_recv` then build results

what do you think ?